### ISSUE TITLE
Error when linting CSS string with cache enabled.

### ISSUE THREAD

**Issue by olegskl** [stylelint/stylelint#2492]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

An error is thrown when trying to lint a CSS string while `cache` option is set to `true.`

> Which rule, if any, is this issue related to?

Not applicable.

> What CSS is needed to reproduce this issue?

Any CSS string.

> What stylelint configuration is needed to reproduce this issue?
```json
{
  "code": ".foo {}",
  "codeFilename": "foo.css",
  "cache": true
}
```

> Which version of stylelint are you using?

`7.10.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

```js
const stylelint = require('stylelint');

stylelint.lint({
  code: '.foo {}',
  codeFilename: 'foo.css',
  cache: true,
  config: {rules: []}
}).catch(console.error);
```

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No.

> What did you expect to happen?

Lint should work. Not sure about caching (probably should be ignored?).

> What actually happened (e.g. what warnings or errors you are getting)?

```shell
TypeError: Cannot read property 'reconcile' of undefined
    at prepareReturnValue (...\node_modules\stylelint\lib\standalone.js:174:16)
    at stylelint._lintSource.then.catch.then.stylelintResult (...\node_modules\stylelint\
lib\standalone.js:90:14)
```

**Comment by evilebottnawi** [comment]
@olegskl thanks for issue :+1:

**Comment by sergesemashko** [comment]
thanks @olegskl, I'll take care of it.

**Comment by davidtheclark** [comment]
Closed by #2494.
